packages/markdown/__tests__/flavored-parsers.test.js

```diff
@@ -0,0 +1,40 @@
+const unified = require('unified');
+const remarkParse = require('remark-parse');
+const rehypeSanitize = require('rehype-sanitize');
+
+const parseCallouts = require('../processor/parse/flavored/callout');
+const parseCodeTabs = require('../processor/parse/flavored/code-tabs');
+const options = require('../processor/options.json').markdownOptions;
+
+const sanitize = { attributes: [], tagNames: [] };
+const process = (text, opts = options) =>
+  text &&
+  unified()
+    .use(remarkParse, opts)
+    .data('settings', { position: false })
+    .use([parseCallouts.sanitize(sanitize), parseCodeTabs.sanitize(sanitize)])
+    .use(rehypeSanitize)
+    .parse(text);
+
+describe('Parse ReadMe-Flavored Markdown Syntax', () => {
+  it('Callouts', () => {
+    const text = `> ℹ️ Info Callout
+    >
```

Comment:
This is just how eslint autoformatted my code, but that said... I _believe_ string literals in JS trim preceding whitespace (up to the indentation level of the given line.) So that string gets interpreted as one of our custom callouts, not a code block. Which, to answer your question, is really just a fancy blockquote!